cartoframes/datasets.py

```diff
@@ -41,6 +41,14 @@ def upload(self, with_lonlat=None, if_exists='fail'):
 
         return self
 
+    def delete(self):
+        if self.exists():
```

Comment:
We can save the `self.exists()` query if we use `_drop_table_query` method (which actually checks for existence). 

The only problem is that we could not return True or False, but I'd expect delete to just return None and in case something went wrong I'll get the corresponding CartoException from SQL API.

Reply:
:+1: I have forgotten the method. I prefer to use it, although we can not return a Boolean result